crates/indexer/src/bits/token_account.rs

```diff
@@ -0,0 +1,50 @@
+use chrono::offset::Local;
+use indexer_core::{
+    db::{
+        insert_into,
+        models::TokenAccount as TokenAccountModel,
+        select,
+        tables::{metadatas, token_accounts},
+    },
+    prelude::*,
+};
+use spl_token::state::Account as TokenAccount;
+
+use crate::{prelude::*, Client};
+
+pub fn process(client: &Client, pubkey: Pubkey, token_account: TokenAccount) -> Result<()> {
+    let mint = token_account.mint.to_string();
+    let owner = token_account.owner.to_string();
+    let pubkey = pubkey.to_string();
+    let db = client.db()?;
+    let now = Local::now().naive_utc();
```

Comment:
```suggestion
    let mint = token_account.mint.to_string();
    let db = client.db()?;
```

Move some of the string conversions after the short-circuit arm.